src/styles/price.css

```diff
@@ -0,0 +1,14 @@
+.price {
+  margin-top: 24px;
```

Comment:
as long as it is block it should not contain external geometry, move it to card__price